Sources/WebUI/WebViewProxy.swift

```diff
@@ -137,16 +149,7 @@ public final class WebViewProxy: ObservableObject {
     /// ```
     @discardableResult
     public func evaluateJavaScript(_ javaScriptString: String) async throws -> Any? {
-        guard let webView else { return nil }
-        return try await withCheckedThrowingContinuation { continuation in
-            webView.wrappedValue.evaluateJavaScript(javaScriptString) { result, error in
-                if let error {
-                    continuation.resume(throwing: error)
-                } else {
-                    continuation.resume(returning: result)
-                }
-            }
-        }
+        try await webView?.wrappedValue.evaluateJavaScript(javaScriptString)
```

Comment:
This asynchronous method has the potential to crash the app when executing a JavaScript method that returns no value, such as `console.log`. Consequently, the method cannot be used in WebUI.

Reply:
I did not know about that behavior.
I will think of a way to implement it.